cli/src/main/java/io/zell/zdb/LogPrintCommand.java

```diff
@@ -12,18 +12,30 @@
 import java.util.concurrent.Callable;
 import picocli.CommandLine.Command;
 import picocli.CommandLine.Model.CommandSpec;
+import picocli.CommandLine.Option;
 import picocli.CommandLine.Spec;
 
 @Command(name = "print", description = "Print's the complete log to standard out")
 public class LogPrintCommand implements Callable<Integer> {
 
   @Spec private CommandSpec spec;
 
+  @Option(
+      names = {"-d", "--dot"},
+      description = "Print's the complete log in dot format, which can be consumed by graphviz")
+  private boolean dotToggle;
```

Comment:
🔧 My personal preference would be a `-f` and `--format` option that allows switching between the default and dot formats. This would allow easily expanding the number of formats in the future.

Reply:
Makes sense to me :) I added the -f flag